code/modules/clothing/glasses/_glasses.dm

```diff
@@ -573,3 +573,38 @@
 	desc = "Lookin' cool."
 	icon_state = "phantom_glasses"
 	inhand_icon_state = "phantom_glasses"
+
+/obj/item/clothing/glasses/dealmaker
+	name = "dealmakers"
+	desc = "Why be the little sponge who hates its $4.99 life when you can be a BIG SHOT?"
```

Comment:
needs the brackets from the original quote, if possible

Reply:
yeah, problem is that byond uses brackets for putting vars in text, so im not quite sure how to do it

Reply:
escape the var brackets with \

Reply:
cool beans